tsl/test/sql/merge_compress.sql

```diff
@@ -29,7 +29,7 @@ INSERT INTO target (series_id, value, partition_column)
 -- compress chunks
 SELECT count(compress_chunk(c.schema_name|| '.' || c.table_name))
   FROM _timescaledb_catalog.chunk c, _timescaledb_catalog.hypertable ht where
-    c.hypertable_id = ht.id and ht.table_name = 'target' and c.compressed_chunk_id IS NULL;
+    c.hypertable_id = ht.id and ht.table_name = 'target' and c.status & 1 = 0;
```

Comment:
Since the concept of "is a chunk that can can be compressed" has changed (which seems to be the reason to why you changed all these expressions) it might make sense to add a function for this purpose. That would avoid having to change these expressions each time we change the notion of what it means that a chunk is "compressable".

Reply:
That check is redundant in most cases anyways, like here where its all new uncompressed chunks. I'd rather opt for removing it altogether.